scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 621,
-  MAX_SIZE: 654,
+  MAX_SIZE: 659,
```

Comment:
Do we know the reason for the size increase? Is there any way it can be mitigated?

Reply:
Not precisely. I assume it's increased because of the new code added, because the biggest bundle size is Russian at 655KB

Reply:
So it's a 1KB increase caused by the latest changes to `psammead-bulleted-list`? I think in this case it is justified but we need to look for opportunities to reduce bundle size as we go along in future, currently it's steadily growing and we are just bumping this value without giving it much thought.

Reply:
the process we use for monitoring this doesn't give much visibility as nobody updates this file from one PR to the next - it only ever gets updated when the CI requires it, so we only see it change for the straws that break the camel's back

Reply:
I totally agree, very open to ideas for how we can improve the process. Just asking the question to raise awareness and make sure people are considering the justification for increasing the limit.